### PR TITLE
Ftr/dev 5616/non covid page updates

### PR DESCRIPTION
**Pull request by marcyheld**
**High level description:**

Non-COVID profile page edits for language change from "Response" to "Spending"

**Technical details:**

Edited labels for Global Nav, Advanced Seach, COVID badge + tooltip, and Award Summary pages
(homepage edits in separate PR for DEV-5113, modal edits in separate PR for DEV-5618)

**JIRA Ticket:**
DEV-5616 is a sub-task of DEV-5594
[DEV-5594](https://federal-spending-transparency.atlassian.net/browse/DEV-5594)

**Mockup:**
Various mockups for each screen are linked in the ticket A/C above

The following are ALL required for the PR to be merged:

Author:
- [x] Linked to this PR in JIRA ticket
- [x] Scheduled demo including Design/Testing/Front-end OR Provided instructions for testing in JIRA and PR `if applicable`
- [x] Verified cross-browser compatibility: Chrome, Safari, Firefox, Internet Explorer, Edge
- [`N/A`] Verified mobile/tablet/desktop/monitor responsiveness
- [`N/A`] Verified that this PR does not create any *new* accessibility issues (via Axe Chrome extension)
- [`N/A`] Added Unit Tests for methods in Container Components, reducers, helper functions, and models `if applicable`
- [`N/A`] All `componentWillReceiveProps` and `componentWillMount` in relevant child/parent components/containers replaced with [future compatible life-cycle methods](https://reactjs.org/blog/2018/03/27/update-on-async-rendering.html) per [JIRA item 3277](https://federal-spending-transparency.atlassian.net/browse/DEV-3277)
- [`N/A`] [API contract](https://github.com/fedspendingtransparency/usaspending-api/tree/dev/usaspending_api/api_contracts) updated `if applicable`
- [`N/A`] [Component Library Integration Status Report](https://github.com/fedspendingtransparency/data-act-documentation/blob/data-transparency-ui/frontend_apps/component-library-integration-status.md) updated `if applicable`

Reviewer(s):
- [`N/A`] Design review complete `if applicable`
- [`N/A`] [API #1234](https://github.com/fedspendingtransparency/usaspending-api/pull/1234) merged concurrently `if applicable`
- [`N/A`] Code review complete

### CODE REVIEW

File: src/js/components/award/shared/awardAmountsSection/charts/AwardAmountsChart.jsx [hunk 1]
```diff
@@ -92,7 +92,7 @@ const buildNormalProps = (awardType, data, hasFileC) => {
                         rawValue: data._fileCObligated,
                         denominatorValue: data._totalObligation,
                         value: data.fileCObligatedAbbreviated,
-                        text: 'COVID-19 Response Obligations Amount',
+                        text: 'COVID-19 Obligated Amount',
```
Comment (MaxwellKendall) [inline src/js/components/award/shared/awardAmountsSection/charts/AwardAmountsChart.jsx:95]: are we sure we don't want spending obligated amount or something?

File: src/js/containers/search/topFilterBar/TopFilterBarContainer.jsx [hunk 2]
```diff
@@ -195,7 +195,7 @@ export class TopFilterBarContainer extends React.Component {
         }
         if (selected) {
             filter.code = 'defCodes';
-            filter.name = 'Disaster Emergency Fund (DEF) Codes';
+            filter.name = 'Disaster Emergency Fund Codes (DEFC)';
```
Comment (MaxwellKendall) [inline src/js/containers/search/topFilterBar/TopFilterBarContainer.jsx:198]: Should this be plural `Codes` or `Code` singular?

File: src/js/components/award/shared/awardAmountsSection/charts/AwardAmountsChart.jsx (hunk 1 above)
Comment (marcyheld) [reply src/js/components/award/shared/awardAmountsSection/charts/AwardAmountsChart.jsx:95]: This language has been cleared w Ross and Andrew - for consistency with the other labels in the chart and with the Advanced Search columns we went with "Obligated" and "Outlayed," and we figured that adding "Spending" in the label would be redundant

Comment (marcyheld) [reply src/js/components/award/shared/awardAmountsSection/charts/AwardAmountsChart.jsx:95]: When we removed "Response" from the Advanced Search columns we also kept "Spending" out of those headings as well

**Pull request merged**